src/plugins/lvm-dbus.c

```diff
@@ -453,6 +453,14 @@ static gchar** get_existing_objects (const gchar *obj_prefix, GError **error) {
     return ret;
 }
 
+
+/**
+ * get_object_path
```

Comment:
I think you need to add a trailing colon `:` character... But then, these are only static private functions.

Reply:
Fixed, I added the docstrings mostly because this isn't the first I was looking at those functions wondering whether it is safe to pass `NULL` for `error` or not. So now I don't need to wonder next time :)